code/game/atoms_movable.dm

```diff
@@ -390,7 +390,7 @@
 
 // harderforce is for things like lighting overlays which should only be moved in EXTREMELY specific sitations.
 /atom/movable/proc/forceMove(atom/destination,var/no_tp=0, var/harderforce = FALSE)
-	var/old_loc = loc
+	var/turf/old_loc = loc
```

Comment:
Why?

Reply:
Because https://travis-ci.org/vgstation-coders/vgstation13/builds/279050728

Reply:
Fuck, that should be atom, not turf.